[2025-10-14T07:10:40.284+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: events_analytics_pipeline.dbt_docs_generate scheduled__2025-10-14T06:00:00+00:00 [queued]>
[2025-10-14T07:10:40.287+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: events_analytics_pipeline.dbt_docs_generate scheduled__2025-10-14T06:00:00+00:00 [queued]>
[2025-10-14T07:10:40.288+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-10-14T07:10:40.292+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): dbt_docs_generate> on 2025-10-14 06:00:00+00:00
[2025-10-14T07:10:40.296+0000] {standard_task_runner.py:60} INFO - Started process 779 to run task
[2025-10-14T07:10:40.298+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'events_analytics_pipeline', 'dbt_docs_generate', 'scheduled__2025-10-14T06:00:00+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/events_pipeline_dag.py', '--cfg-path', '/tmp/tmpvxw6spga']
[2025-10-14T07:10:40.301+0000] {standard_task_runner.py:88} INFO - Job 95: Subtask dbt_docs_generate
[2025-10-14T07:10:40.321+0000] {task_command.py:423} INFO - Running <TaskInstance: events_analytics_pipeline.dbt_docs_generate scheduled__2025-10-14T06:00:00+00:00 [running]> on host 7fd364930f15
[2025-10-14T07:10:40.349+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='events_analytics_pipeline' AIRFLOW_CTX_TASK_ID='dbt_docs_generate' AIRFLOW_CTX_EXECUTION_DATE='2025-10-14T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-14T06:00:00+00:00'
[2025-10-14T07:10:40.349+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-14T07:10:40.350+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/event_gen && /home/***/.local/bin/dbt docs generate --profiles-dir /opt/***/.dbt --project-dir /opt/***/event_gen']
[2025-10-14T07:10:40.356+0000] {subprocess.py:86} INFO - Output:
[2025-10-14T07:10:40.902+0000] {subprocess.py:93} INFO - [0m07:10:40  Running with dbt=1.10.13
[2025-10-14T07:10:40.982+0000] {subprocess.py:93} INFO - [0m07:10:40  Registered adapter: clickhouse=1.9.4
[2025-10-14T07:10:41.107+0000] {subprocess.py:93} INFO - [0m07:10:41  Found 4 models, 30 data tests, 1 source, 599 macros
[2025-10-14T07:10:41.109+0000] {subprocess.py:93} INFO - [0m07:10:41
[2025-10-14T07:10:41.109+0000] {subprocess.py:93} INFO - [0m07:10:41  Concurrency: 4 threads (target='dev')
[2025-10-14T07:10:41.109+0000] {subprocess.py:93} INFO - [0m07:10:41
[2025-10-14T07:10:41.408+0000] {subprocess.py:93} INFO - [0m07:10:41  Building catalog
[2025-10-14T07:10:41.452+0000] {subprocess.py:93} INFO - [0m07:10:41  Catalog written to /opt/***/event_gen/target/catalog.json
[2025-10-14T07:10:42.854+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-14T07:10:42.865+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=events_analytics_pipeline, task_id=dbt_docs_generate, execution_date=20251014T060000, start_date=20251014T071040, end_date=20251014T071042
[2025-10-14T07:10:42.873+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-14T07:10:42.879+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
